.rubocop.yml

```diff
@@ -1,16 +1,503 @@
 require:
   - rubocop-performance
-  - rubocop-rspec
   - rubocop-rails
+  - rubocop-rspec
+
+inherit_from: .rubocop_todo.yml
 
 AllCops:
   TargetRubyVersion: 3.2
   DisplayCopNames: true
   SuggestExtensions: false
-  NewCops: disable
   Exclude:
-    - bin/**
-    - vendor/bundle/**/*
+    - bin/**/*
+    - coverage/**/*
+    - db/**/*
+    - log/**/*
+    - node_modules/**/*
+    - postgres-data/**/*
+    - public/**/*
+    - solr_conf/**/*
+    - spec/fixtures/**/*
+    - tmp/**/*
+    - vendor/**/*
```

Comment:
this alone will greatly speed up running rubocop.   Thank you!

Reply:
Yep. Took the runtime from 20s down to 10s.